user_docs/en/changes.t2t

```diff
@@ -130,6 +130,7 @@ Users of Poedit 1 are encouraged to update to Poedit 3 if they want to rely on e
 - The speech text is no longer updated when the mouse moves in the Speech Viewer. (#15952, @hwf1324)
 - NVDA will again switch back to browse mode when closing combo boxes with ``escape`` or ``alt+upArrow`` in Firefox or Chrome. (#15653)
 - Arrowing up and down in combo boxes in iTunes will no longer inappropriately switch back to browse mode. (#15653)
+- Removed redundant files/folders from the installation directory. (#16068, @hwf1324)

```

Comment:
I don't think this needs a change log entry, it won't really be noticed by users

```suggestion
```